src/overlays/actors/ovl_En_M_Thunder/z_en_m_thunder.c

```diff
@@ -173,7 +173,7 @@ void func_80A9F408(EnMThunder* this, PlayState* play) {
     }
 
     if (player->unk_858 >= 0.1f) {
-        func_800AA000(0.0f, (s32)(player->unk_858 * 150.0f) & 0xFF, 2, (s32)(player->unk_858 * 150.0f) & 0xFF);
+        Rumble_Request(0.0f, (s32)(player->unk_858 * 150.0f) & 0xFF, 2, (s32)(player->unk_858 * 150.0f) & 0xFF);
```

Comment:
I wonder if an `u8` would be better here

Reply:
Turns out it matches without the & 0xFF or a u8 cast